### PR TITLE
Update Fortigate RestAPI procedure

### PR DESCRIPTION
**Pull request by omercier**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md [hunk 1]
```diff
@@ -5,57 +5,77 @@ title: Fortinet Fortigate Rest API
...
-| System   | Net-Fortinet-Fortigate-System-Restapi   | Contrôle l'utilisation système des VDOM (processeur, mémoire, sessions)               | X       |
+Le connecteur apporte les modèles de service suivants
+(classés selon le modèle d'hôte auquel ils sont rattachés) :
+
+<Tabs groupId="sync">
+<TabItem value="Net-Fortinet-Fortigate-Restapi-custom" label="Net-Fortinet-Fortigate-Restapi-custom">
+
+| Alias    | Modèle de service                              | Description                              |
+|:---------|:-----------------------------------------------|:-----------------------------------------|
+| Health   | Net-Fortinet-Fortigate-Health-Restapi-custom   | Contrôle l'état des santé du firewall    |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md:24]:
```suggestion
| Health   | Net-Fortinet-Fortigate-Health-Restapi-custom   | Contrôle l'état de santé du firewall    |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md [hunk 2]
```diff
@@ -5,57 +5,77 @@ title: Fortinet Fortigate Rest API
...
+| System   | Net-Fortinet-Fortigate-System-Restapi-custom   | Contrôle l'utilisation système des VDOM  |
+
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **Net-Fortinet-Fortigate-Restapi-custom** est utilisé.
+
+</TabItem>
+<TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">
+
+| Alias | Modèle de service                        | Description                                            |
+|:------|:-----------------------------------------|:-------------------------------------------------------|
+| Ha    | Net-Fortinet-Fortigate-Ha-Restapi-custom | Contrôle l'utilisation système des members du cluster  |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md:35]:
```suggestion
| Ha    | Net-Fortinet-Fortigate-Ha-Restapi-custom | Contrôle l'utilisation système des membres du cluster  |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md [hunk 3]
```diff
@@ -64,138 +84,339 @@ Il apporte les modèles de services suivants :
...
-    --port='443' \
-    --proto='https' \
-    --api-token='mytoken' \
-    --verbose
+	--plugin=network::fortinet::fortigate::restapi::plugin \
+	--mode=system \
+	--hostname='10.0.0.1' \
+	--port='443' \
+	--proto='https' \
+	--access-token=''  \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md:269]: token?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md [hunk 4]
```diff
@@ -64,138 +84,339 @@ Il apporte les modèles de services suivants :
...
-    active sessions: 0
-checking vdom 'SSTRN'
-    cpu load: 5.00 %
-    memory used: 12.00 %
-    active sessions: 6559
-checking vdom 'root'
-    cpu load: 2.00 %
-    memory used: 4.00 %
-    active sessions: 228
+OK: cpu load: 87 % memory used: 15 % active sessions: 99 | '*vdoms*~cpu.utilization.percentage'=87%;;;0;100'*vdoms*~memory.usage.percentage'=15%;;;0;100'*vdoms*~sessions.active.count'=99;;;0;
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md:283]: output is very different

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md [hunk 5]
```diff
@@ -64,138 +84,339 @@ Il apporte les modèles de services suivants :
...
+| EXTRAOPTIONS   | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                        | --verbose               |             |
+
+</TabItem>
+<TabItem value="Licenses" label="Licenses">
+
+| Macro           | Description                                                                                                                                                | Valeur par défaut       | Obligatoire |
+|:----------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------------|:-----------:|
+| FILTERNAME      | Filter licenses by name (can be a regexp)                                                                                                                  |                         |             |
+| UNIT            | Select the unit for expires threshold. May be 's' for seconds,'m' for minutes, 'h' for hours, 'd' for days, 'w' for weeks. Default is seconds              |                         |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md:228]:
```suggestion
| UNIT            | Select the time unit for thresholds. May be 's' for seconds,'m' for minutes, 'h' for hours, 'd' for days, 'w' for weeks. Default is seconds              |                         |             |
```

File: pp/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md [hunk 6]
```diff
@@ -5,57 +5,76 @@ title: Fortinet Fortigate Rest API
...
+| System        | Net-Fortinet-Fortigate-System-Restapi-custom   | Check VDOM systems    |
+
+> The services listed above are created automatically when the **Net-Fortinet-Fortigate-Restapi-custom** host template is used.
+
+</TabItem>
+<TabItem value="Not attached to a host template" label="Not attached to a host template">
+
+| Service Alias | Service Template                         | Service Description           |
+|:--------------|:-----------------------------------------|:------------------------------|
+| Ha            | Net-Fortinet-Fortigate-Ha-Restapi-custom | Check cluster members system  |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md:34]: Check cluster members system?

File: pp/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md [hunk 7]
```diff
@@ -65,138 +84,339 @@ It brings the following service templates:
 To control your Fortinet Fortigate, the Rest API must be configured.
 E.g: https://docs.fortinet.com/document/fortigate/7.2.1/administration-guide/399023/rest-api-administrator
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md:85]:
```suggestion
See: https://docs.fortinet.com/document/fortigate/7.2.1/administration-guide/399023/rest-api-administrator
```

File: pp/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md [hunk 8]
```diff
@@ -65,138 +84,339 @@ It brings the following service templates:
...
+| EXTRAOPTIONS   | Any extra option you may want to add to the command (E.g. a --verbose flag). All options are listed [here](#available-options)                                                        | --verbose               |             |
+
+</TabItem>
+<TabItem value="Licenses" label="Licenses">
+
+| Macro           | Description                                                                                                                                                | Default value           | Mandatory   |
+|:----------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------------|:-----------:|
+| FILTERNAME      | Filter licenses by name (can be a regexp)                                                                                                                  |                         |             |
+| UNIT            | Select the unit for expires threshold. May be 's' for seconds,'m' for minutes, 'h' for hours, 'd' for days, 'w' for weeks. Default is seconds              |                         |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md:230]:
```suggestion
| UNIT            | Select the time unit for thresholds. May be 's' for seconds,'m' for minutes, 'h' for hours, 'd' for days, 'w' for weeks. Default is seconds              |                         |             |
```

File: pp/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md [hunk 9]
```diff
@@ -65,138 +84,339 @@ It brings the following service templates:
...
-    --port='443' \
-    --proto='https' \
-    --api-token='mytoken' \
-    --verbose
+	--plugin=network::fortinet::fortigate::restapi::plugin \
+	--mode=system \
+	--hostname='10.0.0.1' \
+	--port='443' \
+	--proto='https' \
+	--access-token=''  \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md:269]: token?

File: pp/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md [hunk 10]
```diff
@@ -65,138 +84,339 @@ It brings the following service templates:
...
-    active sessions: 0
-checking vdom 'SSTRN'
-    cpu load: 5.00 %
-    memory used: 12.00 %
-    active sessions: 6559
-checking vdom 'root'
-    cpu load: 2.00 %
-    memory used: 4.00 %
-    active sessions: 228
+OK: cpu load: 87 % memory used: 15 % active sessions: 99 | '*vdoms*~cpu.utilization.percentage'=87%;;;0;100'*vdoms*~memory.usage.percentage'=15%;;;0;100'*vdoms*~sessions.active.count'=99;;;0;
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md:283]: output is very dfferent

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md (hunk 3 above)
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md:269]:
```suggestion
	--access-token='xxxxxx'  \
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md (hunk 4 above)
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md:283]:
```suggestion
OK: All vdom systems are ok | 'ABS#cpu.utilization.percentage'=0.00%;;;0;100 'ABS#memory.usage.percentage'=0.00%;;;0;100 'ABS#sessions.active.count'=155;;;0; 'ADV#cpu.utilization.percentage'=0.00%;;;0;100 'ADV#memory.usage.percentage'=1.00%;;;0;100 'ADV#sessions.active.count'=553;;;0; 'BGN#cpu.utilization.percentage'=0.00%;;;0;100 'BGN#memory.usage.percentage'=0.00%;;;0;100 'BGN#sessions.active.count'=244;;;0; 'LHE#cpu.utilization.percentage'=0.00%;;;0;100 'LHE#memory.usage.percentage'=0.00%;;;0;100 'LHE#sessions.active.count'=100;;;0; 'MED#cpu.utilization.percentage'=3.00%;;;0;100 'MED#memory.usage.percentage'=11.00%;;;0;100 'MED#sessions.active.count'=6280;;;0; 'MIC#cpu.utilization.percentage'=0.00%;;;0;100 'MIC#memory.usage.percentage'=5.00%;;;0;100 'MIC#sessions.active.count'=3244;;;0; 'MLC#cpu.utilization.percentage'=0.00%;;;0;100 'MLC#memory.usage.percentage'=0.00%;;;0;100 'MLC#sessions.active.count'=431;;;0; 'PRN#cpu.utilization.percentage'=0.00%;;;0;100 'PRN#memory.usage.percentage'=0.00%;;;0;100 'PRN#sessions.active.count'=0;;;0; 'SSTRN#cpu.utilization.percentage'=5.00%;;;0;100 'SSTRN#memory.usage.percentage'=12.00%;;;0;100 'SSTRN#sessions.active.count'=6559;;;0; 'root#cpu.utilization.percentage'=2.00%;;;0;100 'root#memory.usage.percentage'=4.00%;;;0;100 'root#sessions.active.count'=228;;;0;
```

File: pp/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md (hunk 6 above)
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md:34]:
```suggestion
| Ha            | Net-Fortinet-Fortigate-Ha-Restapi-custom | Check cluster health  |
```

File: pp/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md (hunk 9 above)
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md:269]:
```suggestion
	--access-token='xxxxxx'  \
```

File: pp/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md (hunk 10 above)
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/network-fortinet-fortigate-restapi.md:283]:
```suggestion
OK: All vdom systems are ok | 'ABS#cpu.utilization.percentage'=0.00%;;;0;100 'ABS#memory.usage.percentage'=0.00%;;;0;100 'ABS#sessions.active.count'=155;;;0; 'ADV#cpu.utilization.percentage'=0.00%;;;0;100 'ADV#memory.usage.percentage'=1.00%;;;0;100 'ADV#sessions.active.count'=553;;;0; 'BGN#cpu.utilization.percentage'=0.00%;;;0;100 'BGN#memory.usage.percentage'=0.00%;;;0;100 'BGN#sessions.active.count'=244;;;0; 'LHE#cpu.utilization.percentage'=0.00%;;;0;100 'LHE#memory.usage.percentage'=0.00%;;;0;100 'LHE#sessions.active.count'=100;;;0; 'MED#cpu.utilization.percentage'=3.00%;;;0;100 'MED#memory.usage.percentage'=11.00%;;;0;100 'MED#sessions.active.count'=6280;;;0; 'MIC#cpu.utilization.percentage'=0.00%;;;0;100 'MIC#memory.usage.percentage'=5.00%;;;0;100 'MIC#sessions.active.count'=3244;;;0; 'MLC#cpu.utilization.percentage'=0.00%;;;0;100 'MLC#memory.usage.percentage'=0.00%;;;0;100 'MLC#sessions.active.count'=431;;;0; 'PRN#cpu.utilization.percentage'=0.00%;;;0;100 'PRN#memory.usage.percentage'=0.00%;;;0;100 'PRN#sessions.active.count'=0;;;0; 'SSTRN#cpu.utilization.percentage'=5.00%;;;0;100 'SSTRN#memory.usage.percentage'=12.00%;;;0;100 'SSTRN#sessions.active.count'=6559;;;0; 'root#cpu.utilization.percentage'=2.00%;;;0;100 'root#memory.usage.percentage'=4.00%;;;0;100 'root#sessions.active.count'=228;;;0;
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**